src/CmsAdmin/DbController.php

```diff
@@ -34,7 +34,6 @@ class DbController extends Controller
      */
     public function dumpAction()
     {
-        ob_end_clean();
```

Comment:
ufam że tak ma być :D

Reply:
Tak, już od dość dawna przeniesione jest to na warstwę generowania szablonu (tego ala smarty), więc wcześniej nigdy nie było ob_start();